packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -247,7 +247,7 @@ export const richTextFormatButtonIconStyle = (theme: Theme, isSelected: boolean)
 export const editBoxRichTextEditorStyle = (): RichTextEditorStyleProps => {
   return {
     minHeight: '2.25rem',
-    maxHeight: '2.25rem'
```

Comment:
The old edit box doesn't change the height. Is it expected that it will increase the height for the rich text one?

Reply:
> The old edit box doesn't change the height. Is it expected that the height will be updated for the rich text one?

The old edit box starts with a shorter height and as content grows, it gets taller till the max-height. I think we can stick with this behaviour?

Reply:
I see, for some reason, I thought that send box can change height but not edit box. Probably it was updated or I missed something...